src/config-render.js

```diff
@@ -0,0 +1,22 @@
+function renderConfig(options, mappedConfig) {
+    const { moduleType } = options;
+    let moduleDefinition = `// This file was automatically generated at ${(new Date()).toISOString()}\n`;
+
+    if (moduleType === 'node') {
```

Comment:
@VitaliyR,
Please note, that although command line parser defines default value of `node`, the exported `buildConfig` function could be run without an options. The main file of this npm is `"main": "./src/index.js"`.

Therefore, instead of using command line binary, user can do:
```
const singleConfig = require('single-config');
singleConfig.buildConfig(inputFile, outputFile)
```
without passing the third `options` parameter.

The code prior this change assumed that options can be empty or not defined, therefore it first set the default value:
```
let moduleType = 'node';
```